admin/tabs/settings-config.php

```diff
@@ -0,0 +1,195 @@
+        <?php
+    }
+
+    public function main_column() {
+        global $wpdb;
+        $setting = get_option( 'dt_setup_wizard_config' );
+        ?>
+        <!-- Box -->
+        <table class="widefat striped">
+          <thead>
+          <tr>
+            <th>Settings Config</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <td>
+              <form name="settingsConfig" onsubmit="settingsConfigSubmit(event)">
+                <label for="config">JSON Config</label>
+
+                <textarea id="config" name="config" class="auto-expand" data-min-rows="3" ><?php echo( $setting );?></textarea>
+
+                <button type="submit">Submit</button>
+              </form>
+            </td>
+          </tr>
+          </tbody>
+        </table>
+        <br>
+        <!-- End Box -->
+        <?php
+    }
+
+    public function right_column() {
+        $sample = json_decode(
+            '{
+          "steps": [{
+            "name": "Basic Settings",
+            "description": "Enter basic blog settings",
+            "config": {
+                "options": [{
+                    "key": "blogname",
+                    "value": "M2M Journey Tools"
+                }, {
+                    "key": "blogdescription",
+                    "value": " "
+                }, {
+                    "key": "admin_email",
+                    "value": "xg-web-admin@xtendglobal.org"
+                }]
+            }
+        }, {
+            "name": "Custom Roles",
+            "description": "Ensure any custom roles are added.",
+            "config": {
+                "options": [{
+                    "key": "dt_custom_roles",
+                    "value": {
+                        "custom_plugin_admin": {
+                            "description": "This role only has the ability to access D.T\'s wp-admin, manage D.T, and edit plugins, notably to manage plugin integrations. Role has no access to post data.",
+                            "label": "Plugin Admin",
+                            "slug": "custom_plugin_admin",
+                            "capabilities": [
+                                "access_disciple_tools",
+                                "edit_plugins",
+                                "activate_plugins",
+                                "manage_dt",
+                                "manage_options",
+                                "read"
+                            ]
+                        }
+                    }
+                }]
+            }
+        }, {
+            "name": "Mapping",
+            "description": "Create a geocoding key at [https://console.cloud.google.com/apis/credentials?project=disciple-tools-geocoding](https://console.cloud.google.com/apis/credentials?project=disciple-tools-geocoding). Key should have permissions for\n\n - Geocoding API\n - Maps Javascript API\n - Places API\n\nEnter the created API Key below.",
```

Comment:
Let's remove the link in this description since it has our own project ID in it.